aom/src/main/java/com/nedap/archie/xml/adapters/DateTimeXmlAdapter.java

```diff
@@ -21,9 +19,12 @@ public TemporalAccessor unmarshal(String stringValue) {
 
     @Override
     public String marshal(TemporalAccessor value) {
-        if(value instanceof LocalDateTime || value instanceof ZonedDateTime || value instanceof OffsetDateTime) {
-            return value.toString();
```

Comment:
I did not check that specifically. But I saw that the [DateTimeSerializer.java](https://github.com/openEHR/archie/pull/446/files#diff-c58fd4f77d2a490a3a7c00dfb792fd188ee625cf2e9fe731d579b3b194a58de4) that you changed in that PR did the same. I might be wrong, but it seemed to me that the formatting will now be done in the same way for Json and xml.

Besides, I removed the first block, because the last line `return DateTimeSerializerFormatters.ISO_8601_DATE_TIME.format(value);` ended up not being hit by any of the current tests at all. I can keep it in, but I'm not sure what test case would hit that line.

Reply:
I have checked and the date time format seems to cover all cases.